src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -50,6 +50,7 @@ internal sealed class AD7DebugSession : DebugAdapterBase, IDebugPortNotify2, IDe
         private int m_nextContextId = 1;
 
         private Dictionary<string, IDebugPendingBreakpoint2> m_functionBreakpoints;
+        private Dictionary<string, IDebugPendingBreakpoint2> m_instructionBreakpoints;
```

Comment:
Is `InstructionReference` the correct key? Or should it be some combination of `InstructionReference`+`Offset`

Reply:
It should either be InstructionReference + Offset, or you could resolve it first, and make this a map of ULONG.